pkg/storaged/vgroup-details.jsx

```diff
@@ -218,7 +221,10 @@
                 <CardHeader actions={{
                     actions: (
                         <>
-                            <StorageButton onClick={() => vgroup_rename(client, vgroup)}>{_("Rename")}</StorageButton>
+                            <StorageButton excuse={ vgroup.MissingPhysicalVolumes && vgroup.MissingPhysicalVolumes.length > 0
+                                ? _("A volume group with missing physical volumes can not be renamed.")
+                                : null }
+                                           onClick={() => vgroup_rename(client, vgroup)}>{_("Rename")}</StorageButton>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17226-20230912-094641-c7f396be-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)